src/plugins/hotkeys/hotkeys.ts

```diff
@@ -0,0 +1,106 @@
+import './volume-bar.css'
+import type { pluginParameter } from 'shared/assets/types/types.js'
+
+/**
+ * Vlitejs Volume bar plugin
+ * @module Vlitejs/plugins/Hotkeys
+ */
+export default class Hotkeys {
+	player: any
+
+	providers = ['html5', 'youtube', 'dailymotion', 'vimeo']
+	types = ['video', 'audio']
+
+	/**
+	 * @constructor
+	 * @param options
+	 * @param options.player Player instance
+	 */
+	constructor({ player }: pluginParameter) {
+		this.player = player
+
+		this.player.onKeydown = this.onKeydown.bind(this)
+	}
+
```

Comment:
Plugin needs to have a `init` function for initialization.

It is called here

https://github.com/vlitejs/vlite/blob/8a45ec788be3d4843ffbe23c75fd13af9424e585/src/core/vlite.ts#L156-L161

https://github.com/vlitejs/vlite/blob/8a45ec788be3d4843ffbe23c75fd13af9424e585/src/plugins/plugin.ts#L85

Reply:
And fyi, a `onReady` function is also available, triggered when the player is ready. Not sure if this one is useful for this plugin

https://github.com/vlitejs/vlite/blob/8a45ec788be3d4843ffbe23c75fd13af9424e585/src/core/player.ts#L204-L206

Reply:
Imo `onReady` function isn't needed.